lxd/db/instances.go

```diff
@@ -674,7 +667,7 @@ SELECT storage_pools.name FROM storage_pools
   JOIN instances ON instances.name=storage_volumes_all.name
   JOIN projects ON projects.id=instances.project_id
  WHERE projects.name=?
-   AND storage_volumes_all.node_id=?
+   AND (storage_volumes_all.node_id=? OR storage_volumes_all.node_id IS NULL AND storage_pools.driver IN ("ceph", "cephfs"))
```

Comment:
This is looking up instances so node_id cannot be NULL in this case.

Reply:
It can be NULL. When an instance is created, `CreateStoragePoolVolume` is called which only sets a node if the driver is not remote.